parser/src/test_utils.rs

```diff
@@ -124,12 +125,16 @@ impl<R> ClearSourceRefs for Expression<R> {
             | Expression::UnaryOperation(_, _)
             | Expression::BinaryOperation(_, _)
             | Expression::IndexAccess(_, _)
+            | Expression::FieldAccess(_, _)
             | Expression::FunctionCall(_, _)
             | Expression::FreeInput(_, _)
             | Expression::IfExpression(_, _) => {}
             Expression::BlockExpression(_, b) => b.clear_source_refs(),
             Expression::MatchExpression(_, m) => m.clear_source_refs(),
             Expression::LambdaExpression(_, l) => l.clear_source_refs(),
+            Expression::StructExpression(_, f) => {
```

Comment:
This is only needed for stuff that has patterns. "Ordinary" expressions are already handled in line 116/117